NDB.Covid19/NDB.Covid19.iOS/Views/SelftestOption/SelftestOptionViewController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using Foundation;
+using NDB.Covid19.iOS.Utils;
+using NDB.Covid19.Utils;
+using NDB.Covid19.ViewModels;
+using UIKit;
+using NDB.Covid19.iOS.Views.AuthenticationFlow;
+
+namespace NDB.Covid19.iOS.Views.SelftestOption
+{
+    public partial class SelftestOptionViewController : BaseViewController, IUIAccessibilityContainer
+    {
+        public string RegisterTestTitle = "RegistertestTitleText";
+        public SelftestOptionViewController(IntPtr handle) : base(handle)
+        {
+        }
+
+        public static SelftestOptionViewController GetSelftestOptionViewController()
+        {
+            UIStoryboard storyboard = UIStoryboard.FromName("SelftestOption", null);
+            SelftestOptionViewController vc = storyboard.InstantiateInitialViewController() as SelftestOptionViewController;
+            vc.ModalPresentationStyle = UIModalPresentationStyle.FullScreen;
+            return vc;
+        }
+
+        SelftestRegistrationViewModel _viewmodel;
+        UIViewController _authViewController;
+        UIButton _ContinueButton;
+
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+            _viewmodel = new SelftestRegistrationViewModel();
+            SetTexts();
+            SetupStyling();
+            //ContinueWithMSISBtn.TouchUpInside += ContinueButtonTapped;
```

Comment:
Why is this commented out?

Reply:
i forgot to delete it, but it is not supposed to be in the code. I have deleted it now.